subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -538,6 +539,7 @@ static void ctrl_disable(struct bt_mesh_light_ctrl_srv *srv)
 
 	/* Clear transient state: */
 	atomic_and(&srv->flags, FLAGS_CONFIGURATION);
+	atomic_clear_bit(&srv->flags, FLAG_AMBIENT_LUXLEVEL_SET);
```

Comment:
Separate `atomic_clear_bit` is not needed, `atomic_and` will clear ambient luxlevel flag.